src/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -70,7 +70,7 @@ static boolean testIntegrityLevel(Context cx, Object o, INTEGRITY_LEVEL level) {
             if (Boolean.TRUE.equals(desc.get("configurable"))) return false;
 
             if (level == INTEGRITY_LEVEL.FROZEN
-                    && desc.isDataDescriptor(desc)
+                    && ScriptableObject.isDataDescriptor(desc)
```

Comment:
Is there something missing in this PR? AFAICS ScriptableObject doesn't have a static isDataDescriptor method

Reply:
They are static since https://github.com/mozilla/rhino/commit/c275c35f160ef7b035002c67c3540ba317964351

Reply:
Sry, missed that